tests/Jellyfin.Server.Implementations.Tests/Sorting/AiredEpisodeOrderComparerTests.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Emby.Server.Implementations.Sorting;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Entities.Movies;
+using MediaBrowser.Controller.Entities.TV;
+using Xunit;
+
+namespace Jellyfin.Server.Implementations.Tests.Sorting
+{
+    public class AiredEpisodeOrderComparerTests
+    {
+        [Theory]
+        [ClassData(typeof(EpisodeTestData))]
+        public void Test1(BaseItem x, BaseItem y, int expected, bool err)
```

Comment:
I'm not sure what a good test name would be here, but I don't think "Test1" is descriptive enough

Reply:
That is embarrassing, updated it to be named a bit better.